ExtendCanvas/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  ~
+  -->
+
+<resources>
+    <string name="extend_canvas_app_name">Erweiterte Leinwand</string>
```

Comment:
It can work, but we should go for a common approach for all sample-names, since others use the English name